src/Chinchilla.Specifications/Serializers/DefaultMessageTypeFactorySpecification.cs

```diff
@@ -1,36 +1,36 @@
-﻿using Chinchilla.Serializers;
-using Machine.Fakes;
-using Machine.Specifications;
+﻿// using Chinchilla.Serializers;
+// using Machine.Fakes;
+// using Machine.Specifications;
 
-namespace Chinchilla.Specifications.Serializers
-{
-    public class DefaultMessageTypeFactorySpecification
-    {
-        [Subject(typeof(DefaultMessageTypeFactory))]
-        public class in_general : WithSubject<DefaultMessageTypeFactory>
-        {
-            It should_not_have_cached_factory_by_default = () =>
-                Subject.HasCachedFactory(typeof(IMagicMessage)).ShouldBeFalse();
-        }
+// namespace Chinchilla.Specifications.Serializers
+// {
+//     public class DefaultMessageTypeFactorySpecification
```

Comment:
still need to figure out what to do with this, if we're using newtonsoft i think we can elide it as that can be taken care of by the serializer.

Reply:
This is fixed using netstandard 1.6. See comment below.

Reply:
Oops, need to uncomment this.